docs/source/concept_guides/quantization.mdx

```diff
@@ -1,5 +1,256 @@
+  - `Z` is called the zero-point, it is the `int8` value corresponding to the value `0` in the `float32` realm. This is
+  important to be able to represent exactly the value `0` because it is used everywhere throughout machine learning
+  models.
+
+
+The quantized value `x_q` of `x` in `[a, b]` can be computed as follows:
+
+```
+x_q = round(x/S + Z)
+```
+
+And `float32` values outside of the `[a, b]` range are clipped to the closest representable value, so for any
+floating-point number `x`:
+
+```
+x_q = clip(round(x/S + Z), round(a/S + Z), round(b/S + Z))
+
+```
+
+This is called the *affine quantization sheme* because the mapping from `[a, b]` to `int8` is an affine one.
+
+A common special case of this scheme is the *scale quantization scheme*, where we consider a symmetric range of float values `[-a, a]`.
+In this case the integer space is usally `[-127, 127]`, meaning that the `-128` is opted out of the regular `[-128, 127]` signed `int8` range.
+The reason being that having both ranges symmetric allows to have `Z = 0`. While one value out of the 256 representable
+values is lost, it can provide a speedup since a lot of addition operations can be skipped.
+
+**Note**: To learn how the quantization parameters `S` and `Z` are computed, you can read the
+[Quantization and Training of Neural Networks for Efficient Integer-Arithmetic-Only Inference](https://arxiv.org/abs/1712.05877)
+paper, or [Lei Mao's blog post](https://leimao.github.io/article/Neural-Networks-Quantization) on the subject.
+
+
+#### Number of quantization parameters
+
+Depending on the accuracy / latency trade-off you are targetting you can play with the granularity of the quantization parameters:
+
+- Quantization parameters can be computed on a *per-tensor* basis, meaning that one pair of `(S, Z)` will be used per
+tensor.
+- Quantization parameters can be computed on a *per-channel* basis, meaning that it is possible to store a pair of
+`(S, Z)` per element along one of the dimensions of a tensor. For example for a tensor of shape `[N, C, H, W]`, having
+*per-channel* quantization parameters for the second dimension would result in having `C` pairs of `(S, Z)`. While this
+can give a better accuracy, it requires more memory.
+
+#### Calibration
+
+The section above described how quantization from `float32` to `int8` works, but one question
+remains: how is the `[a, b]` range of `float32` values determined? That is where calibration comes in to play.
+
+Calibration is the step during quantization where the `float32` ranges are computed. For weights it is quite easy since
+the actual range is know at *quantization-time*. But it is less clear for activations, and different approaches exist:
+
+1. Post training dynamic quantization: the range for each activation is computed at *runtime*. While this gives great
+results without too much work, it can be a bit slower than static quantization because the range is computed each time.
```

Comment:
```suggestion
results without too much work, it can be a bit slower than static quantization because over the overhead introduced of computing the range each time.
```